src/ServiceV2/ResettingService.php

```diff
@@ -174,4 +186,9 @@ public function processSendingPasswordResetSms(string $phone): ?User
 
         return null;
     }
+
+    private function isPublicResetPasswordRequest(): bool
+    {
+        return null === $this->getUser();
```

Comment:
I am not sure this is the good check. You can try to reset your password, but still have a session available, in this cas, you will have a bug